applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -364,6 +363,20 @@ build_bridge(#state{endpoints=Endpoints
        | kz_api:default_headers(Q, <<"call">>, <<"command">>, ?APP_NAME, ?APP_VERSION)
       ]).
 
+-spec maybe_ignore_early_media(kz_term:api_binary(), kapi_offnet_resource:req()) -> kz_term:api_binary().
+maybe_ignore_early_media(CCVs, OffnetReq) ->
+    RequireIgnoreEarlyMedia=kz_json:get_ne_binary_value(<<"Require-Ignore-Early-Media">>, CCVs, <<"false">>),
+    ResourceIgnoreEarlyMedia=kapi_offnet_resource:ignore_early_media(OffnetReq, <<"false">>),
+    case {RequireIgnoreEarlyMedia, ResourceIgnoreEarlyMedia} of
```

Comment:
I would prefer something that better captures the intent here.

It appears there is an ordering of preference: `true > consume > ring_ready > false`.

But it is unclear whether `RequireIEM` or `ResourceIEM` has an ordering. For instance, if `RequireIEM` is `consume` and `ResourceIEM` is `true`, it seems `ResourceIEM` takes precedence (because `true > consume`), which seems surprising. I would expect `RequireIEM` to take precedence (it is "required" after all).

Is it a naming issue? Should `Require` be `Requested`?

Could you take another attempt at writing this function and see if you can come up with something that's more clear on what's happening, and if not, add a nice big comment explaining what's happening? :)